dist/colors.scss

```diff
@@ -488,109 +488,101 @@ $blk-235: #060606;
-  --calcite-ui-yellow-2: #{$ui-yellow-2};
-  --calcite-ui-yellow-3: #{$ui-yellow-3};
-  --calcite-ui-red-1: #{$ui-red-1};
-  --calcite-ui-red-2: #{$ui-red-2};
-  --calcite-ui-red-3: #{$ui-red-3};
+  --calcite-ui-brand: #{$ui-brand};
+  --calcite-ui-brand-hover: #{$ui-brand-hover};
+  --calcite-ui-brand-press: #{$ui-brand-press};
   --calcite-ui-background: #{$ui-background};
   --calcite-ui-foreground-1: #{$ui-foreground-1};
   --calcite-ui-foreground-2: #{$ui-foreground-2};
   --calcite-ui-foreground-3: #{$ui-foreground-3};
   --calcite-ui-text-1: #{$ui-text-1};
   --calcite-ui-text-2: #{$ui-text-2};
   --calcite-ui-text-3: #{$ui-text-3};
+  --calcite-ui-text-inverse: #{$ui-text-inverse};
+  --calcite-ui-text-link: #{$ui-text-link};
   --calcite-ui-border-1: #{$ui-border-1};
   --calcite-ui-border-2: #{$ui-border-2};
   --calcite-ui-border-3: #{$ui-border-3};
-  --calcite-ui-border-4: #{$ui-border-4};
-  --calcite-ui-border-5: #{$ui-border-5};
+  --calcite-ui-border-input: #{$ui-border-input};
+  --calcite-ui-info: #{$ui-info};
+  --calcite-ui-success: #{$ui-success};
+  --calcite-ui-warning: #{$ui-warning};
+  --calcite-ui-danger: #{$ui-danger};
+  --calcite-ui-danger-hover: #{$ui-danger-hover};
+  --calcite-ui-danger-press: #{$ui-danger-press};
 }
 
 // Dark theme as CSS variable set
 @mixin calcite-theme-dark() {
-  --calcite-ui-blue-1: #{$ui-blue-1-dark};
-  --calcite-ui-blue-2: #{$ui-blue-2-dark};
-  --calcite-ui-blue-3: #{$ui-blue-3-dark};
-  --calcite-ui-green-1: #{$ui-green-1-dark};
-  --calcite-ui-green-2: #{$ui-green-2-dark};
-  --calcite-ui-green-3: #{$ui-green-3-dark};
-  --calcite-ui-yellow-1: #{$ui-yellow-1-dark};
-  --calcite-ui-yellow-2: #{$ui-yellow-2-dark};
-  --calcite-ui-yellow-3: #{$ui-yellow-3-dark};
-  --calcite-ui-red-1: #{$ui-red-1-dark};
-  --calcite-ui-red-2: #{$ui-red-2-dark};
-  --calcite-ui-red-3: #{$ui-red-3-dark};
+  --calcite-ui-brand: #{$ui-brand-dark};
+  --calcite-ui-brand-hover: #{$ui-brand-hover-dark};
+  --calcite-ui-brand-press: #{$ui-brand-press-dark};
   --calcite-ui-background: #{$ui-background-dark};
   --calcite-ui-foreground-1: #{$ui-foreground-1-dark};
   --calcite-ui-foreground-2: #{$ui-foreground-2-dark};
   --calcite-ui-foreground-3: #{$ui-foreground-3-dark};
   --calcite-ui-text-1: #{$ui-text-1-dark};
   --calcite-ui-text-2: #{$ui-text-2-dark};
   --calcite-ui-text-3: #{$ui-text-3-dark};
+  --calcite-ui-text-inverse: #{$ui-text-inverse-dark};
+  --calcite-ui-text-link: #{$ui-text-link-dark};
   --calcite-ui-border-1: #{$ui-border-1-dark};
   --calcite-ui-border-2: #{$ui-border-2-dark};
   --calcite-ui-border-3: #{$ui-border-3-dark};
-  --calcite-ui-border-4: #{$ui-border-4-dark};
-  --calcite-ui-border-5: #{$ui-border-5-dark};
+  --calcite-ui-border-input: #{$ui-border-input-dark};
+  --calcite-ui-info: #{$ui-info-dark};
+  --calcite-ui-success: #{$ui-success-dark};
+  --calcite-ui-warning: #{$ui-warning-dark};
+  --calcite-ui-danger: #{$ui-danger-dark};
```

Comment:
In regards to @macandcheese's comment.
We've been talking about a "destructive" action.
I think it might be good to have a variable for the status red. And a variable for what we'd use for the interactive element (i.e. button). 

I think making these the same might turn out to be brittle as designs progress.

Reply:
Danger === Destructive here, right now the button is just "color=red" but we'd change it to "danger" to match the nomenclature here. imo having a second red variable would be confusing.

Reply:
Right now, we have a set of vars essentially for our notices/warnings and set of vars for a destructive/danger button...and one of those vars is used for both. And in order to know the difference, you need our tribal knowledge.

Seems like keeping those two sets of vars would actually bring clarity.

For instance

```
// Just for the warning/notice
--calcite-ui-info
--calcite-ui-success
--calcite-ui-warning
--calcite-ui-danger

// For interactive
--calcite-ui-destructive
--calcite-ui-destructive-hover
--calcite-ui-destructive-press
```

This also decouples the color for the destructive (or whatever that gets called).
If it's the same var for the interactive (i.e. button), borders on notices/warnings, possibly error state inputs...this creates a brittle system.

Seems like we may also want to expose the interactive "destructive" (or whatever we call it) var for theming.

Reply:
@jrowlingson @TheBlueDog @paulcpederson 
Please chime in if you have any suggestions about this.

Reply:
What’s “brittle” about that? 

We are going to update the buttons currently referred to as “red” to “danger”, so it matches the nomenclature and prevents the need for another variable. 

I don’t see a use case for having those two reds be different colors? If a user needs to change the accent bar red for whatever reason, maybe it happens to be their brand color, this means it’s updated in all places it’s used.

Reply:
Sorry if I was unclear. 
I'm not suggesting they be different colors.
I'm proposing that there be a separate variable for the red used for statuses.

In this PR, the variable for status red and for the button are the same.
Our intention for the status colors was to retain them for stuff like calcite-notice. They don't have interactive states.

The button colors do have states.
Seems like we'd wanna make a clear distinction between those sets of vars.

It's brittle because it requires that the red for a danger or error notification be the same as that of a destructive button.

Reply:
Disagree - the intention is to have the idle “red” button color be the same as the “status accent red”, just like it currently is now.

We’ve never actually used “destructive” - we can now use “danger” for both the button and the accent color with no extra thinking needed.

Reply:
Sorry if I was unclear. I'm not advocating for using "destructive" in our variable names.

Reply:
I think the intention is a design intention. And that's cool. I dig that.

I think that it might be helpful in our vars to make a clear distinction between the variables used for notice/status status colors and the three colors used for the "danger" button.

Right now, there's a non-obvious overlap between the status color vars and the interactive/button vars.

Reply:
IMO what’s proposed in the PR is good

Reply:
This is more accurate to what I'm suggesting.
```
/**
*  For the alerts, notifications, and perhaps input borders
* stuff that has statuses
*/
--calcite-ui-info
--calcite-ui-success
--calcite-ui-warning
--calcite-ui-error    // can be the same as danger

/**
* For buttons (and perhaps other interactive elements)
*/
--calcite-ui-danger    // can be the same as error but could also be themed
--calcite-ui-danger-hover
--calcite-ui-danger-press

--calcite-ui-brand
--calcite-ui-brand-hover
--calcite-ui-brand-press
```

Reply:
Again... disagree. Error is not applicable as a name most of the time, that should stay danger as above. I really don’t see the need for the extra variable. Especially in the example above. Now you just have -error and -danger reflecting the same underlying color.

Reply:
Error notifications and alerts seems like a normal thing right?

Reply:
Sure, but there are plenty of times it’s not necessarily an error. Consider a “red border modal” - that’s just a container that may prompt to to delete something forever, it’s not an error - so danger is more applicable and still works for error. Just like bootstrap.

Reply:
All right. I'll leave this for another day.